src/iocore/net/QUICNetVConnection.cc

```diff
@@ -502,10 +511,8 @@ QUICNetVConnection::is_handshake_completed() const
 void
 QUICNetVConnection::net_read_io(NetHandler *nh, EThread *lthread)
 {
-  if (quiche_conn_is_readable(this->_quiche_con)) {
-    SCOPED_MUTEX_LOCK(lock, this->mutex, this_ethread());
-    this->handleEvent(QUIC_EVENT_PACKET_READ_READY, nullptr);
-  }
+  SCOPED_MUTEX_LOCK(lock, this->mutex, this_ethread());
+  this->handleEvent(QUIC_EVENT_PACKET_READ_READY, nullptr);
```

Comment:
Checking `quiche_conn_is_readable` was wrong because the function checks whether there are readable *streams*. There may be something need to be processed on the connection level.